build.gradle

```diff
@@ -82,6 +82,15 @@ artifactory {
 artifactoryPublish.skip = true
 
 subprojects {
+    repositories {
+        mavenCentral()
+        maven { url "http://maven.sk89q.com/repo/" }
+        maven { url "http://repo.maven.apache.org/maven2" }
```

Comment:
Remove this repo? It's just `mavenCentral()` again.

Reply:
Oh, yeah - no idea how this got added